leptos_reactive/src/trigger.rs

```diff
@@ -0,0 +1,187 @@
+#![forbid(unsafe_code)]
+#![allow(missing_docs)]
+
+use crate::{
+    diagnostics,
+    diagnostics::*,
+    node::NodeId,
+    runtime::{with_runtime, RuntimeId},
+    Scope, ScopeProperty, SignalGet, SignalSet, SignalUpdate,
+};
+
+pub trait TriggerDirty {
+    fn dirty(&self);
```

Comment:
I wonder whether there's a better name for `dirty` here. `.trigger()`? `.mark_dirty()`? It wasn't initially obvious to me whether this meant `.is_dirty()` or `.mark_dirty()`, since I don't often use "dirty" as a verb on its own.